packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.spec.tsx

```diff
@@ -367,6 +367,22 @@ describe("horizontal overflow", (): void => {
 
         expect(renderedWithImages.instance().state.itemsHeight).toBe(0);
     });
+    test("should call `onHorizontalOverflowChange` in the `onLoad` method if prop exists", () => {
```

Comment:
nit: we can remove reference to the method itself. 

"should call `onHorizontalOverflowChange` when initialized when a `onHorizontalOverflowChange` prop is passed" or something like it. 

I don't think you need to simulate load either. You should be able to just render it with the prop and get the values. 

Simulate is somewhat of a hack in this sense. Does this not work without `simulate`?